x-pack/packages/kbn-elastic-assistant/impl/assistant/conversations/conversation_settings/conversation_settings.tsx

```diff
@@ -220,7 +219,7 @@ export const ConversationSettings: React.FC<ConversationSettingsProps> = React.m
             compressed
             conversation={selectedConversation}
             isEditing={true}
-            isDisabled={selectedConversation == null}
+            isDisabled={isDisabled}
```

Comment:
Ty! 👍 

`isDisabled` is passed in as `isDisabled={selectedConversation == null}`, so equivalence!